docs/Reference/Pantheon-API-Methods.md

```diff
@@ -1029,6 +1196,24 @@ You can interact with contracts using [eth_sendRawTransaction or eth_call](../Us
     }
     ```
 
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block {number call (data : {from : \"0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b\", to: \"0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13\", data :\"0x12a7b914\"}){data status}}}"}' http://localhost:8547/graphql
```

Comment:
It looks like this request is a bit different than the RPC one too.

Reply:
I'm not sure how to make these equivalent without spending some time on it.  Will raise a separate JI and may or may not get to it before the release on Wednesday.

Reply:
Raised https://pegasys1.atlassian.net/browse/PAN-2713